services/code_hosting/git_hub_service.rb

```diff
@@ -41,6 +41,12 @@ def repos
       client.repos({}, query: { sort: "asc" })
     end
 
+    # Does the client with the associated credentials have access to the specified repo?
+    # @repo [String] Repo URL as string
+    def access_to_repo?(repo: nil)
+      client.repository?(repo.sub("https://github.com/",""))
```

Comment:
yeah, in this case, it's not a `repo` it's just a string. You could either rename it `git_url` or pass in a `repo_config` item and extract the `git_url` from there.

Reply:
if i name the parent `repo_url` i'll still have to use that name correct? Can the child change the name of the param?

Reply:
you'll need to use `repo_url` if you change it to that. The child cannot change the name of a named param.

Reply:
done